crates/matrix-sdk-ui/src/timeline/tests/polls.rs

```diff
@@ -297,9 +280,9 @@ impl TestTimeline {
         original_id: &EventId,
         content: UnstablePollStartContentBlock,
     ) {
-        let content =
-            ReplacementUnstablePollStartEventContent::new(content, original_id.to_owned());
-        let event_content = AnyMessageLikeEventContent::UnstablePollStart(content.into());
+        let event_content = UnstablePollStartEventContent::from(
+            ReplacementUnstablePollStartEventContent::new(content, original_id.to_owned()),
+        );
```

Comment:
Note to self: There's also a `EventFactory::poll_edit` function, but it can't easily be used without also changing the signature (thus the callers) here.